iree/compiler/Dialect/Shape/Conversion/test/shape_to_shapex.mlir

```diff
@@ -45,6 +45,40 @@ func @f(%arg0: tensor<?xf32>, %index: i32) {
   return
 }
 
+// -----
+// shape.get_extent
+// CHECK-LABEL: func @f
+func @f(%arg0: tensor<?xf32>) {
+  %c0 = constant 0 : index
+  // CHECK: [[SHAPE:%.+]] = shapex.get_ranked_shape %arg0 : tensor<?xf32> -> !shapex.ranked_shape<[?]>
```

Comment:
Suggest `%[[SHAPE:.+]]` style for consistency.

Reply:
Done.